src/goto-programs/goto_convert.cpp

```diff
@@ -1258,6 +1258,7 @@ void goto_convertt::convert_switch(
 
   goto_programt tmp_cases;
 
+  size_t case_number=1;
```

Comment:
this would be great, but will be another issue, when implementing (loop) conditional pretty-printing. For now it should be ok to use ordinals, independent on the actual value of the switch variable for the case.